deployment/docker/requirements.txt

```diff
@@ -15,4 +15,6 @@ flask-restful==0.3.6
 PyYAML
 numpy
 Flask-Script
-simplekml==1.3.0
\ No newline at end of file
+simplekml==1.3.0
+psycopg2
```

Comment:
use specific versions please

Reply:
added specific versions for requirements.